cannon/mipsevm/multithreaded/mips.go

```diff
@@ -247,7 +246,10 @@ func (m *InstrumentedState) syscallGetRandom(a0, a1 uint64) (v0, v1 uint64) {
 	randDataMask <<= (arch.WordSizeBytes - byteCount) * 8
 	randDataMask >>= targetByteIndex * 8
 	newMemVal := (memVal & ^randDataMask) | (randomWord & randDataMask)
+
+	m.memoryTracker.TrackMemAccess(effAddr)
```

Comment:
nit: While this is correct, it's more intuitive to me to `TrackMemAccess` before calling `GetWord` on L230. Because it's signaling an intent to generate the witness to prove a read value in memory. Doing this after we've already read the memory word seems backwards.

Reply:
I was trying to group related operations, but you're right trackMem is more closely related to `GetWord` - will update